docs/maps/mapBubble-series.md

```diff
@@ -0,0 +1,11 @@
+Map Bubble
+==========
+
+MapBuble series is a combination of the standard bubble chart series with the possibility of applying to specific parts of the map.
```

Comment:
Typo: MapBuble -> MapBubble

Reply:
Hmm, when I read "is a combination", I expect the whole sentence in a format `It is a combination of A and B.` (missing `and` keyword).

Maybe something like this?: `The MapBubble series is a combination of the standard bubble chart and map, where each bubble point is displayed over a specific map point shape.`